investigator/investigator/missing_package/investigate_missing_package.py

```diff
@@ -37,30 +36,11 @@
 @register_handler(MissingPackageMessage().topic_name, ["v1"])
 async def parse_missing_package(package: Dict[str, Any], openshift: OpenShift, graph: GraphDatabase):
     """Process a missing package message from package-update producer."""
-    repositories = graph.get_adviser_run_origins_all(
-        index_url=package["index_url"], package_name=package["package_name"], count=None, distinct=True,
-    )
+    # TODO call kebechet with the missing package name, which would call kebechet on the individual
```

Comment:
Here should be calling kebechet administrator as well right?

Reply:
We would need implement way to handle this on kebechet side, and then comeback to this.

Reply:
Did you create issue in Kebechet? If not please add its :) Thanks @saisankargochhayat :)

Reply:
I would deactivate the metric then for now :)

Reply:
@pacospace thanks I have commented it out for now.